frontend/components/dashboard/DashboardControls.vue

```diff
@@ -90,7 +90,7 @@ const manageButtons = computed<MenuBarEntry[] | undefined>(() => {
     return [ {
       dropdown: true,
       highlight: true,
-      items: buttons,
+      items: buttons.reverse(), // In the dropdown we want the button sorting to be reversed
```

Comment:
nitpick: creating layout or changing layout in `script`-part is confusing. 
The layout should be defined through composition of `components`.

Reply:
for simplicity I leave it here. ;-)